components/brave_shields/common/brave_shield_switches.h

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_SHIELDS_COMMON_BRAVE_SHIELD_SWITCHES_H_
+#define BRAVE_COMPONENTS_BRAVE_SHIELDS_COMMON_BRAVE_SHIELD_SWITCHES_H_
+
+namespace brave_shields {
+namespace switches {
+
+// The following command-line switches allow to override defaults of
+// corresponding shields. ATTENTION: using any of this switch persists
+// a given value in content settings of any loaded user profile. Switches
+// behavior mirrors the behavior of toggles on brave://settings/shields.
+//
+// Example: --set-shields-ads-default=block
+
+constexpr char kShieldsAdsSetDefault[] = "set-shields-ads-default";
```

Comment:
I believe defining these inline causes some issue on windows.

Reply:
do you have any exact examples? Basically this way is suggested by CC and is used across the codebase